src/pages/partner-notification-services/tabs/partner-notification.component.tsx

```diff
@@ -1,21 +1,94 @@
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import EmptyState from '../../../components/empty-state/empty-state.component';
+import EncounterList, {
+  EncounterListColumn,
+  getObsFromEncounter,
+} from '../../../components/encounter-list/encounter-list.component';
+import { deathFormEncounterType_UUID } from '../../../constants';
 
 interface PartnerNotificationListProps {
   patientUuid: string;
 }
 
+const columnsLab: EncounterListColumn[] = [
+  {
+    key: 'Date',
+    header: 'Date',
+    getValue: encounter => {
+      return getObsFromEncounter(encounter, true);
+    },
+  },
+  {
+    key: 'relationshipToIndex',
+    header: 'Relationship to Index',
+    getValue: encounter => {
+      return getObsFromEncounter(encounter, true);
+    },
+  },
+  {
+    key: 'indexType',
+    header: 'Index Type',
+    getValue: encounter => {
+      return getObsFromEncounter(encounter, true);
+    },
+  },
+  {
+    key: 'contactMethod',
+    header: 'Contact Method',
+    getValue: encounter => {
+      return getObsFromEncounter(encounter, true);
+    },
+  },
+  {
+    key: 'contactOutcome',
+    header: 'Contact Outcome',
+    getValue: encounter => {
+      return getObsFromEncounter(encounter, true);
+    },
+  },
+
+  {
+    key: 'actions',
+    header: 'Actions',
+    getValue: encounter => {
+      const baseActions = [
+        {
+          form: { name: '', package: '' },
```

Comment:
Aren't we supposed to return the same form(patient_tracing)? I see you set the from to empty, is this intended? I would default to: `package: 'hiv', name: 'patient_tracing' `

Reply:
Are we?? Because there's a Patient tracing tab already. Thought it was one of those forms we don't have yet but can ask the BAs

Reply:
This is for actions on a particular encounter, you can either decide to view or edit that encounter